services/src/messaging/src/main/java/org/openkilda/messaging/info/ChunkedInfoMessage.java

```diff
@@ -20,21 +20,35 @@
 import static org.openkilda.messaging.Utils.TIMESTAMP;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.ToString;
 
 @Getter
 @ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
 public class ChunkedInfoMessage extends InfoMessage {
 
-    @JsonProperty("next_request_id")
-    private String nextRequestId;
+    @JsonProperty("message_id")
+    private String messageId;
 
-    public ChunkedInfoMessage(@JsonProperty(PAYLOAD) final InfoData data,
-                              @JsonProperty(TIMESTAMP) final long timestamp,
-                              @JsonProperty(CORRELATION_ID) final String correlationId,
-                              @JsonProperty("next_request_id") String nextRequestId) {
+    @JsonProperty("total_messages")
+    private int totalMessages;
+
+    public ChunkedInfoMessage(@JsonProperty(PAYLOAD) InfoData data,
+                              @JsonProperty(TIMESTAMP) long timestamp,
+                              @JsonProperty(CORRELATION_ID) String correlationId,
+                              @JsonProperty("message_id") String messageId,
+                              @JsonProperty("total_messages") int totalMessages) {
+        super(data, timestamp, correlationId);
+        this.messageId = messageId;
+        this.totalMessages = totalMessages;
+    }
+
+    public ChunkedInfoMessage(InfoData data, long timestamp, String correlationId,
+                              int messageIndex, int totalMessages) {
         super(data, timestamp, correlationId);
-        this.nextRequestId = nextRequestId;
+        this.messageId = correlationId + " : " + messageIndex;
```

Comment:
reverse order i.e. messageIndex + ":" + correlationId would be more "usual"... i.e. we are joining correlationIds in this way, so keep same order here looks natural.